test/e2e/data/infrastructure-vsphere-govmomi/main/memory-reservation-locked/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
+patches:
+  - target:
+      kind: VSphereMachineTemplate
+    patch: |-
+      - op: add
+        path: /spec/template/spec/memoryReservationLockedToMax
```

Comment:
may be enough to add this to an existing test to not increase the runtime

Reply:
Would it then make sense to add default variable in env package:
```
var (
	DefaultMemoryReservationLockedToMax = toBoolPtr(true)
)
```

Reply:
let's just add it to one test, not to all :-)

Reply:
I reviewed the code, and it seems that if I want to integrate this with an existing test, like the `hw-upgrade` flavor, I can directly add the `memory-locked-to-max` check in the `hardware_upgrade_test.go` file. This would allow us to remove the `memory_reservation_locked_test.go` file entirely. However, I'm unsure if this is what you meant by "add this to an existing test to not increase the runtime"?